antora-playbook-for-publication.yml

```diff
@@ -1,17 +1,5 @@
----
-#
-# Copyright (c) 2020 Red Hat, Inc.
-# This program and the accompanying materials are made
-# available under the terms of the Eclipse Public License 2.0
-# which is available at https://www.eclipse.org/legal/epl-2.0/
-#
-# SPDX-License-Identifier: EPL-2.0
-#
-
-# Use this Antora Playbook for publication, to build the released version branch.
 site:
```

Comment:
I believe we can remove the antora-playbook-for-publication.yml from the main branch. It will create confusion one day. I believe having it in the publication-builder branch should be enough.